### DISCUSSION TITLE
Missing coordinates in saved .sgy file

### DISCUSSION THREAD

**Discussion by Filipe-Adao** [emanuelhuber/RGPR#83] [Q&A]
Hello,
I successfully processed a GPR profile using RGPR and saved it in the .sgy format using the writeGPR function. I then tried to open the saved file in Geohammer and Prism2 software but there were no coordinates available. Do you know what might be the problem? Thanks in advance.

**Comment by emanuelhuber** [comment]
Hello @Filipe-Adao 

Thank you for your message. I am currently on travel. I will have a look at it in 2 weeks.
Best

Emanuel

**Comment by emanuelhuber** [comment]
@Filipe-Adao 

I have two questions concerning the issue you raised:

1. Does the RGPR object have coordinates? What is the output of `coord(x)`? (`x` is the GPR object)
2. Do Geohammer and Prism2 read segy-coordinates?

Best,
Emanuel
